sessions/2/pavel_shchegolevatykh/main.py

```diff
@@ -0,0 +1,25 @@
+def is_palindrome(word_to_check):
+    word_length = len(word_to_check)
```

Comment:
this check can be simplified to:
reversed = word_to_check[::-1]
if reversed == word_to_check:
  return True

Reply:
Cool feature, thanks. Fixed in the next PR.